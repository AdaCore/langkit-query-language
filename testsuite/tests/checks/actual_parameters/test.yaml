driver: 'checker'
rule_name: 'Actual_Parameters'
project: 'prj.gpr'
rule_arguments:
   actual_parameters.forbidden: '[("actuals.subp1", "info", "|actuals\.info.*")]'
