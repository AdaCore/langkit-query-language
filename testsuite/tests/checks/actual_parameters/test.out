actuals.adb:9:14: rule violation: actual parameter mismatch
9 |       Subp1 (Info => Integer'(Info2));   --  FLAG
  |              ^^^^^^^^^^^^^^^^^^^^^^^

actuals.adb:10:14: rule violation: actual parameter mismatch
10 |       Subp1 (Info => Integer (Info2));   --  FLAG
   |              ^^^^^^^^^^^^^^^^^^^^^^^

